packages/ignite-cli/src/cli/check.js

```diff
@@ -31,7 +31,7 @@ var rnCli = enforceGlobalDependency({
   range: '>=2.0.0',
   which: 'react-native',
   packageName: 'react-native-cli',
-  versionCommand: 'react-native --version',
+  versionCommand: '--version',
```

Comment:
The `versionCommand` now only takes the actual version flag, not the whole command. This is because we find the executable using the new `which` call.